examples/all/all_vendor.odin

```diff
@@ -39,10 +39,10 @@ import TTF        "vendor:sdl2/ttf"
 
 import vk         "vendor:vulkan"
 
-import NS         "vendor:darwin/Foundation"
-import MTL        "vendor:darwin/Metal"
-import MTK        "vendor:darwin/MetalKit"
-import CA         "vendor:darwin/QuartzCore"
+// import NS         "vendor:darwin/Foundation"
+// import MTL        "vendor:darwin/Metal"
+// import MTK        "vendor:darwin/MetalKit"
+// import CA         "vendor:darwin/QuartzCore"
```

Comment:
Did you comment this out intentionally, or was it just for testing?

Reply:
It was very intentional, but not necessary in the long run. The reason is that I was testing for compile errors locally by running:

```
$ odin run examples/all -vet -strict-style; rm all
```

And the `examples/all` package was including a bunch of darwin stuff on my linux machine which resulted in a crap ton of errors. The other PR I have pending also has these lines commented-out.

Reply:
`examples/all` is technically not meant to use `odin build`, only `odin check`.

Reply:
Thanks for the tip!

No more accidental pushing binaries haha